vivisect/qt/funcgraph.py

```diff
@@ -243,6 +258,7 @@ def __init__(self, vw, vwqgui):
         self.addHotKeyTarget('funcgraph:inczoom', self._hotkey_inczoom)
         self.addHotKey('ctrl+-', 'funcgraph:deczoom')
         self.addHotKeyTarget('funcgraph:deczoom', self._hotkey_deczoom)
+        self.addHotKey('f5', 'funcgraph:refresh')
 
```

Comment:
totally dig the idea of a unified refresh mechanism. ( and a hotkey for it ).  Are there any circumstances which should trigger the refresh without needing the user to do it?